cmd/openshift-install/create.go

```diff
@@ -344,17 +344,10 @@ func waitForInitializedCluster(ctx context.Context, config *rest.Config) error {
 				}
 				clusterVersion = cv
 				if cov1helpers.IsStatusConditionTrue(cv.Status.Conditions, configv1.OperatorAvailable) {
-					logrus.Debug("Cluster is initialized")
 					return true, nil
 				}
-				if cov1helpers.IsStatusConditionTrue(cv.Status.Conditions, configv1.OperatorFailing) {
-					logrus.Debugf("Still waiting for the cluster to initialize: %v",
-						cov1helpers.FindStatusCondition(cv.Status.Conditions, configv1.OperatorFailing).Message)
-					return false, nil
-				}
-				if cov1helpers.IsStatusConditionTrue(cv.Status.Conditions, configv1.OperatorProgressing) {
-					logrus.Debugf("Still waiting for the cluster to initialize: %v",
-						cov1helpers.FindStatusCondition(cv.Status.Conditions, configv1.OperatorProgressing).Message)
+				if status := clusterVersionString(cv); status != "" {
```

Comment:
why not drop the global `var clusterVersion *configv1.ClusterVersion` and switch status to `lastErr`, them you can do something like:
```
lastErr = nil
UntilWithSync
   available -> (true, nil)
   !available ->
     lastErr = <why failing or progressing>
     debug lastErr
     (false, nil)
err != nil && err == wait.ErrTimeout
  lastErr -> (lastErr)
  (err)
(nil)
```

Reply:
> why not drop the global `var clusterVersion *configv1.ClusterVersion` and switch status to `lastErr`...

The messages are just strings, so I've pushed b50248e4c -> 48a4d4d57 moving to a `var lastError string`.  How does that look?